web-ui/src/main/resources/catalog/components/viewer/owscontext/OwsContextService.js

```diff
@@ -324,6 +325,13 @@
             firstLoad = false;
           }
         }
+        
+        // broadcast context load
+        $rootScope.$broadcast('owsContextLoaded');
+       
+        deferred.resolve();
```

Comment:
Dummy promise but useful to have loadContext and loadContextByURL being used indistinctly on the source code.

Reply:
I don't think it really makes sense to return a promise here as there is no async logic, but why not. At least could you please update the JSDoc block with a `@return` line? Thanks

Reply:
Also you can just do `$q.when()`